crawler.py

```diff
@@ -81,102 +85,127 @@ def check_and_add_functions_metrics(metrics, f_metrics_a, f_metrics_b):
 
   return metrics
 
-def save_data(metrics):
+def iter_javascript_repositories():
+  url = 'https://api.github.com/search/repositories'
+  for page in range(1, 101):
+    params = dict(
+      q='language:javascript',
+      sort='stars',
+      per_page='100',
+      page=page
+    )
```

Comment:
Lets iterate over predefined list of repositories, i.e.:
```
repos = ['freeCodeCamp/freeCodeCamp',
         'vuejs/vue',
         'twbs/bootstrap',
         'facebook/react',
         'nodejs/node',
         'jquery/jquery']
```
It'll make metrics collection more deterministic.